src/plugins/files/public/components/file_picker/file_picker.tsx

```diff
@@ -84,6 +85,9 @@ const Component: FunctionComponent<InnerProps> = ({ onClose, onDone, onUpload, m
       className="filesFilePicker filesFilePicker--fixed"
       maxWidth="75vw"
       onClose={onClose}
+      css={css`
+        height: 75vw; // Always be max height for the file picker
```

Comment:
Seems to be breaking the modal on mobile. 
Previously it was nicely expanded to the full screen

Reply:
```suggestion
        min-height: 75vh; // Always be max height for the file picker
```

This seems to be making it as intended. 
Also I think there was a typo: `vw` -> `vh`

Reply:
Haha great point, for some reason I often confuse these units even though they are really different 😅

Reply:
> Seems to be breaking the modal on mobile.

Great point, had not considered this, added a media query for this case.